django/chat/templates/chat/components/chat_history_sidebar.html

```diff
@@ -29,5 +29,16 @@
         {% endfor %}
       </ul>
     </div>
+    <div class="p-3">
+      <button type="button"
+              class="btn btn-outline-danger w-100"
+              id="delete-all-chats"
+              hx-target="#chat-history-list"
+              hx-post="{% url 'chat:delete_all_chats' %}"
+              hx-confirm="Are you sure you want to delete all chats? This action is irreversible."
+              hx-confirm-target="#delete-all-chat-modal">
```

Comment:
Not a valid attribute, maybe a holdover from when you were doing a custom modal? Harmless, but you can remove it.